### PR TITLE
Instagram follow  not follow

### LINKED ISSUE #894: Instagram Follow- NotFollow
# Aim

- Create an Instagram chatbot to display people you follow and who do not follow you back.
I am a GSSOC'21 Participant and would like to work on this issue.

### PR DESCRIPTION
**Pull request by nidhivanjare**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

## Have you read the [Contributing Guidelines on Pull Requests](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/CONTRIBUTING.md)?

- [x] Yes
- [ ] No

## Type of change

_Please delete options that are not relevant._

- [x] New feature (non-breaking change which adds functionality)
- [x] Added ream me file

## Checklist:

- [x] My code follows the style guidelines(Clean Code) of this project
- [x] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md`
- [x] My documentation follows [`Template for README.md`](https://github.com/avinashkranjan/Amazing-Python-Scripts/blob/master/Template%20for%20README.md)
- [x] My changes generate no new warnings
- [x] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works.

Output of my code:

(List of people how do not follow back :)

Screenshot 2021-04-21 at 6 22 12 PM

### CODE REVIEW

**Comment by nidhivanjare** [comment]
Hey, Can you please merge this PR ??

File: Instagram Follow- NotFollow/main.py [hunk 1]
```diff
@@ -0,0 +1,92 @@
+from selenium import webdriver
```
Comment (kaustubhgupta) [inline Instagram Follow- NotFollow/main.py:1]: Print the list of followers and non-followers in a more attractive way. Maybe an ascii table with two columns: followers | non followers

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by nidhivanjare** [comment]
working on it !!

File: .vscode/settings.json [hunk 2]
```diff
@@ -0,0 +1,3 @@
+{
```
[the remark this answers is not in the document]
Comment (kaustubhgupta) [reply .vscode/settings.json:1]: This is still left

File: Instagram Follow- NotFollow/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# Instagram Unfollowers Bot
+
+## Install Chrome Driver
+
+- https://chromedriver.storage.googleapis.com/index.html?path=90.0.4430.24/
+
+Install Selenium
```
Comment (kaustubhgupta) [inline Instagram Follow- NotFollow/README.md:7]:
```suggestion
## Install Selenium
```

**Review by kaustubhgupta — CHANGES_REQUESTED** [review]

**Comment by nidhivanjare** [comment]
Done ??

**Review by kaustubhgupta — APPROVED** [review]

**Review by Kushal997-das — APPROVED** [review]
LGTM 👍

**Comment by nidhivanjare** [comment]
Can you merge this PR?

**Review by antrikshmisri — APPROVED** [review]

**Comment by antrikshmisri** [comment]
> Can you merge this PR?

Only PA can merge the PR

**Review by santushtisharma10 — APPROVED** [review]

**Review by avinashkranjan — APPROVED** [review]
LGTM

**Pull request merged**